tests/foreman/cli/test_repositories.py

```diff
@@ -47,3 +49,45 @@ def test_positive_custom_products_disabled_by_default(
     assert rhel_contenthost.subscribed
     product_details = rhel_contenthost.run('subscription-manager repos --list')
     assert 'Enabled:   0' in product_details.stdout
+
+
+def test_positive_disable_rh_repo_with_basearch(module_target_sat, module_entitlement_manifest_org):
+    """Verify that users can disable Red Hat Repositories with basearch
+
+    :id: dd3b63b7-1dbf-4d8a-ab66-348de0ad7cf3
+
+    :steps:
+        1.  You have the Appstream Kicstart repositories release version
+            "8" synced in from the release of RHEL 8
+        2.  hammer repository-set disable --basearch --name --product-id
+            --organization --releasever
+
+
+    :expectedresults: Users can now disable Red Hat repositories with
+        basearch
+
+    :customerscenario: true
+
+    :BZ: 1932486
+    """
+    rh_repo_id = module_target_sat.api_factory.enable_rhrepo_and_fetchid(
+        basearch=DEFAULT_ARCHITECTURE,
+        org_id=module_entitlement_manifest_org.id,
+        product=REPOS['kickstart']['rhel8_aps']['product'],
+        repo=REPOS['kickstart']['rhel8_aps']['name'],
+        reposet=REPOS['kickstart']['rhel8_aps']['reposet'],
+        releasever=REPOS['kickstart']['rhel8_aps']['version'],
+    )
+    repo = module_target_sat.api.Repository(id=rh_repo_id).read()
+    repo.sync(timeout=2000)
```

Comment:
isn't this timeout too much time for kickstart repo? 
```suggestion
    repo.sync(timeout=600)
```